trustfall_core/src/schema/adapter/schema.graphql

```diff
@@ -1,150 +1,167 @@
 schema {
-    query: RootSchemaQuery
+  query: RootSchemaQuery
 }
 
 """
 Query the contents of Trustfall schemas: their vertex types, properties, edges, etc.
 """
 type RootSchemaQuery {
-    """
-    All types of vertices present in this schema.
-    """
-    VertexType: [VertexType!]!
-
-    """
-    The entry point edges at which querying may begin.
-
-    Corresponds to the valid edge names for the `resolve_starting_vertices()`
-    method for adapters over this schema.
-    """
-    Entrypoint: [Edge!]!
+  """
+  All types of vertices present in this schema.
+  """
+  VertexType: [VertexType!]!
+
+  """
+  The entry point edges at which querying may begin.
+
+  Corresponds to the valid edge names for the `resolve_starting_vertices()`
+  method for adapters over this schema.
+  """
+  Entrypoint: [Edge!]!
+
+  Schema: Schema!
```

Comment:
didn't know what the comment should be here so I left it blank but we probably should add one